HW11_3.py

```diff
@@ -0,0 +1,31 @@
+class MyContextManager:
+    def __enter__(self):
+        print("1==========")
+                        
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        print(f'1111 {exc_type}')
+       # value = my_dict[name] 
+      
+        if exc_type == KeyError:
+            print(f'Key {name} does not exist')
+            print("2==========")
+        else:
+            print(f'Phone number is {value}')
+            print("2==========")
```

Comment:
Add here `return True` to avoid raising of exception

Reply:
done